controls/chat/styling.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Styling
+page_title: .NET MAUI Chat Documentation - Styling
+description: Learn how to style the Telerik UI for .NET MAUI Chat control.
+position: 6
+tags: .net maui, telerik .net maui, ui for .net maui, chat, style
+slug: chat-styling
+---
+
+# .NET MAUI Chat Styling
+
+Any change in the appearance of the `RadChat` components depends on the referenced styles. The default templates contains a `RadBorder` control (used to achieve the rounded edges), an Image control (used for the avatar image only for the single and first messages) and Label for the text message itself.
+
+Here are the provided styles that you can use to customize the different parts of the control:
+
+- `MessageImageStyle`(`TargetType Image`)&mdash;Defines the style referred to the send button image.
+
+- `OutgoingMessageImageStyle` (`TargetType Image`)&mdash;Defines the style of the outgoing message image.
+
+- `IncomingBorderStyle` (`TargetType RadBorder`)&mdash;Defines the style of the incoming message border.
+
+- `OutgoingBorderStyle` (`TargetType RadBorder`)&mdash;Defines the style of the outgoing message border.
+
+- `DefaultLabelStyle` (`TargetType Label`)&mdash;Defines the default label style.
+
+- `OutgoingLabelStyle` (`TargetType Label`)&mdash;Defines the style of the label of the outgoing message.
+
+## Customize the Chat Entry
+
+You can customize the `ChatEntry` with implicit style. The example below shows how to customize the chat entry in your Chat control.
+
+```XAML
+<Style TargetType="telerik:ChatEntry">
```

Comment:
can we use code snippets from sdk app here? if there are replace the xaml with code snippet